br/pkg/lightning/restore/restore.go

```diff
@@ -975,6 +975,11 @@ func (rc *Controller) saveStatusCheckpoint(ctx context.Context, tableName string
 	case !common.IsContextCanceledError(err):
 		merger.SetInvalid()
 		rc.errorSummaries.record(tableName, err, statusIfSucceed)
+	case utils.IsRetryableError(err):
```

Comment:
It seems retryable error doesn't contain "connection refused"?

Reply:
Try fixing it with `MessageIsRetryableStorageError`.